<launch>
	<node name="dynamixel_interface_controller_node" pkg="dynamixel_interface" type="dynamixel_interface_controller_node" output="screen">
		<rosparam command="load" file="$(find eagle_mpc_debugger)/config/interface_controller_s500_uam.yaml" />
	</node>
</launch>
